src/test/java/org/broadinstitute/consent/http/db/DatasetDAOTest.java

```diff
@@ -1262,26 +1273,13 @@ private FileStorageObject createFileStorageObject(String entityId, FileCategory
     return fileStorageObjectDAO.findFileById(newFileStorageObjectId);
   }
 
-  private Dataset insertDataset() {
-    User user = createUser();
-    String name = "Name_" + randomAlphanumeric(20);
-    Timestamp now = new Timestamp(new Date().getTime());
-    String objectId = "Object ID_" + randomAlphanumeric(20);
-    DataUse dataUse = new DataUseBuilder().setGeneralUse(true).build();
-    Integer id = datasetDAO.insertDataset(name, now, user.getUserId(), objectId,
-        dataUse.toString(), null);
-    createDatasetProperties(id);
-    return datasetDAO.findDatasetById(id);
-  }
-
-
-  protected void createDatasetProperty(Integer datasetId, String schemaProperty, String value,
-      PropertyType type) {
+  protected void createDatasetProperty(Integer datasetId, String value) {
```

Comment:
Optional: Maybe `createStringDatasetProperty`? I assume it was only used for strings before, but that way it is clear there are other types if we want them.